lektor_i18n.py

```diff
@@ -140,7 +140,7 @@ def merge_pot(from_filenames, to_filename):
         msgcat = locate_executable("msgcat")
         if msgcat is None:
             msgcat = "/usr/bin/msgcat"
-        cmdline = [msgcat, "--use-first"]
+        cmdline = [msgcat, "--use-first", "-F"]
```

Comment:
```suggestion
        cmdline = [msgcat, "-F"]
```
We need to cumulate file positions